.travis.yml

```diff
@@ -1,18 +1,34 @@
-sudo: false
+dist: trusty
+sudo: required
```

Comment:
Do we need this? Builds without sudo complete faster, so I think we should avoid it if we can.

Reply:
Not sure if I can install packages without sudo. I can test.